acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/widgets/AttachmentsSection.java

```diff
@@ -1,10 +1,12 @@
-package uk.nhs.digital.ps.test.acceptance.pages;
+package uk.nhs.digital.ps.test.acceptance.pages.widgets;
```

Comment:
Given we have this in a widgets package now, do we need to include "Section" in the class name? Would just Attachments do, or maybe AttachmentsWidget?

Reply:
The reason for not renaming these classes using more uniformed pattern was that this package contains a mix of classes representing larger and smaller components. For example, 'Attachments' is classified as a larger 'Section' because it comprises header, the 'Add' button and, potentially, a number of individual upload fields. The `PubliclyAccessibleRadioButton` on the other hand, represents much smaller and less complicated component and that's why it got suffix `Button`.

It could be argued, however, that most of them encapsulate more complexity than an individual, atomic 'control' - even the `PubliclyAccessibleRadioButton` is actually a group of radio buttons, each with its own label and a common header - so perhaps it is more fitting to name them more uniformly. Granted, the fact that they are all 'screen widgets' is reflected in the package name but reflecting such qualification in the class name is still useful - at the very least, it helps when searching for them in the IDE, where we target class name rather than a package.

I'll rename them all to be suffixed with `Widget`.

Reply:
Makes sense. Given your reasoning I'd be happy to leave the names as is; happy either way.

Reply:
Renamed.